CONTRIBUTING.md

```diff
@@ -1,9 +1,13 @@
 Nice to see you want to contribute to this project! :+1: :tada:
 Please have a look at this guide to know what you are changing.
 
-As I do not want to duplicate the instructions all over, please **find the common contributors docs here**: https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md
+### **Note:** This project is an add-on from the TinyWebEx modules. 
```

Comment:
Not really, I've fixed it. Also I don't really like using headings (#) in markdown for highlighting…

So maybe?

```suggestion
### Note
This project uses the same guidelines as the TinyWebEx library modules for add-on's. 

```

Note nowadays you can also use proper notes etc. in GitHub's markdown, so we probably want to use that?
https://github.com/orgs/community/discussions/16925

(But anyway, let's adjust the text. Offline-QR-Code is not a TinyWebEx thing in itself, it uses TinyWebEx modules, which is an important technical difference, which could otherwise confuse users.)

Reply:
Sounds good! I didn't know that feature existed in GitHub's markdown now, thanks for letting me know - I tried implementing it according to the link you provided. Here's the most recent [commit with the requested changes](https://github.com/rugk/offline-qr-code/pull/320/commits/1ab75467bb771cf3cf47b4a0461c6a001698e2f0).